presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedQueries.java

```diff
@@ -23,4 +29,93 @@ public TestTpchDistributedQueries()
     {
         super(createQueryRunner());
     }
+
+    @Test
+    public void testExplainAnalyze()
```

Comment:
Please break this big test into a set of single assert test methods with descriptive names.  Or if you really don't want to do that, put in some comments explaining what you are checking with each assert.